Samples/Islands/DrawingIsland/DrawingIslandComponents/VisualItem.h

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+namespace winrt::DrawingIslandComponents::implementation
+{
+    class VisualItem
```

Comment:
Please add a class comment that explains __what__ a VisualItem's primary purpose is for, and why it's not just a Visual. For example, this is how we notify owners when they need to handle a DPI scale change to re-rastersize their text.